pkg/client/userd/systema_proxy/service_proxy.go

```diff
@@ -0,0 +1,88 @@
+package systema_proxy
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net/url"
+
+	"github.com/datawire/dlib/dlog"
+	"github.com/telepresenceio/telepresence/rpc/v2/systema"
+	"github.com/telepresenceio/telepresence/v2/pkg/a8rcloud"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/cliutil"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/userd/trafficmgr"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+// This proxy is a temporary solution to allow the client to get the
+// preferred ingress info from the cloud
+type SystemAProxy struct {
```

Comment:
`systema_proxy.SystemAProxy` breaks the Golang "no stuttering" rule. How about naming the package `systema` and the type `Proxy`? You'd call `systema.NewProxy()` to get a `systema.Proxy`. Then use an import alias to resolve the rpc name collision:
```
rpc "github.com/telepresenceio/telepresence/rpc/v2/systema"
```

Reply:
Why do we need this proxy in the first place? Why can't the intended caller instead call directly to SystemA?